tests/agent_unittests/test_utilization_settings.py

```diff
@@ -118,22 +118,6 @@ def reset(wrapped, instance, args, kwargs):
     return reset
 
 
-@reset_agent_config(INI_FILE_WITHOUT_UTIL_CONF, ENV_WITHOUT_UTIL_CONF)
```

Comment:
I think this test was intended to be the host: None test. But I think we should keep these tests as they do test different things.

Reply:
That's fine. We can move them into the other file then. I think I just saw a place where it was validating env vars and stuck it in there so my bad.